building.rb

```diff
@@ -1,2 +1,30 @@
 class Building
+
+  attr_reader :apartments
+
+  def initialize(args = {})
+    @address = args.fetch(:address)
+    @apartments = args.fetch(:apartments)
+  end
+
+  def total_room_count
+    apartments.inject(0) { |sum, apartments| sum + apartments.room_count }
+  end
+
+  def total_monthly_revenue
+    apartments.inject(0) { |sum, apartments| sum + apartments.monthly_rent }
```

Comment:
same... as below
You iterate over each apartment - "apartments" represent the array.

+    apartments.inject(0) { |sum, apartment| sum + apartment.monthly_rent }